### PR TITLE
Update cuisine

### PR DESCRIPTION
**Pull request by AlertSubject**
more details

### CODE REVIEW

**Comment by LaoshuBaby** [comment]
I don't think so. Most cafe have some other cuisine such as juice or cookies, but the mainly and definitely is coffee_shop.

My opinion is only if this cafe brand choose some special cuisine as their cuisine, then add it.

**Comment by AlertSubject** [comment]
Im currently updating the PR some more
Finally found a full menu in their online ordering website.
They sell a lot of sandwiches, noodles, and soups as well as salads

https://shop.campussuite.de/hamburg/

So i'd update the cuisine tag accordingly

**Comment by LaoshuBaby** [comment]
> Finally found a full menu in their online ordering website. They sell a lot of sandwiches, noodles, and soups as well as salads
> 
> https://shop.campussuite.de/hamburg/
> 
> So i'd update the cuisine tag accordingly

But NSI is not a menu, we don't need to suggest all food it offer, especially for those common cuisine in cafe shop.

**Comment by AlertSubject** [comment]
I feel like the coffe shop tag alone wouldn't represent the chain fairly. A large part of what they sell is sandwiches (based on my last visit)
They also advertised their warm meals in-store 

Would you be fine with "cuisine": "coffee_shop;sandwich"?
Less detailed but still including the important parts

**Comment by LaoshuBaby** [comment]
> Would you be fine with "cuisine": "coffee_shop;sandwich"?
> Less detailed but still including the important parts

Yes, this is accecpetable



> A large part of what they sell is sandwiches (based on my last visit)

Because this is the special part of this coffee shop.

File: data/brands/amenity/cafe.json [hunk 1]
```diff
@@ -624,7 +624,7 @@
         "amenity": "cafe",
         "brand": "Campus Suite",
         "brand:wikidata": "Q111205702",
-        "cuisine": "coffee_shop",
+        "cuisine": "coffee_shop;juice;sandwich;cookie",
```
Comment (LaoshuBaby) [inline data/brands/amenity/cafe.json:627]:
```suggestion
        "cuisine": "coffee_shop;sandwich",
```

**Review by LaoshuBaby — APPROVED** [review]

**Comment by AlertSubject** [comment]
Alright, thanks for your input!

Is there anything I have to do or did you do all the changes?

**Pull request merged**